iroha/src/asset.rs

```diff
@@ -124,7 +124,10 @@ pub mod query {
         fn execute(&self, world_state_view: &WorldStateView) -> Result<QueryResult, String> {
             let assets: Vec<Asset> = world_state_view
                 .read_account(&self.account_id)
-                .ok_or("No account found.")?
+                .ok_or(format!(
+                    "No account with id: {:?} found in the current world state: {:?}.",
```

Comment:
If we debug print the whole World State View wouldn't the output be to big? As I understand transactions and domains will be printed, and there could be many of them.

Reply:
Possibly, but I think we can afford it for now to save time figuring out what is wrong during development and tests failures. When it will become a problem we will remove it.